constants.xml

```diff
@@ -368,6 +368,12 @@
             </entry>
         </exclusion_list>
         <IGNORE_BLOCKCOSIG_CHECK>false</IGNORE_BLOCKCOSIG_CHECK>
+        <microblock_exclusion_list>
```

Comment:
Here is the context of the problem.
https://zilliqa-team.slack.com/archives/C03AYQAAF8E/p1661842114584239
`1664279` block's macroblock 0  was lost. So we need to skip this block while validating persistence, otherwise, the `validator.cpp` will complain.

Reply:
This list will remain forever until we somehow feed dummy blocks into persistence to replace those.